config.js

```diff
@@ -45,6 +45,17 @@ var config = {
       url: 'https://test-insight.bitpay.com:443',
     },
   },
+  pushNotificationsOpts: {
+    templatePath: './lib/templates',
+    defaultLanguage: 'en',
+    defaultUnit: 'btc',
+    subjectPrefix: '',
+    publicTxUrlTemplate: {
```

Comment:
Is this required? Can you include a link to the block explorer into the notification text?